less/mixins.less

```diff
@@ -1 +1,35 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.button(@width, @bg-color){
+    width: @width;
+    background-color: @bg-color;
+    border-radius: 30px;
+    display: flex;
+}
+//
+
+
+.destination{
+    .btn{
+        .button(82%, #0a5c69);
+        vertical-align: middle;
+        height: 45px;   
+        padding: 15px;
+        font-size: 1.5rem;
+        &:hover {
+            background: fade(#0a5c69, 70%);
+            transition-duration: 1.4s;
+            a{
+                transition-duration: 1.5s;
+                color: #000000;
+                font-weight: bolder;
+                font-size: 1.9rem;
+            }
+          }
+        a{
+            color: white;
+            margin: 0 auto;
+            text-decoration: none;
+            letter-spacing: 2px;
+        }
+    }
+}
```

Comment:
This folder shouldn't have any styles in it. You can use your mixins in any of the files as long as they are imported in the Index.less file before them

this could have gone better in the home-page.less file